### DISCUSSION TITLE
"[Warning:SubmersedVR] Game was not started in VR. Don't load and apply SubmersedVR patches." "[Info   : Unity Log] OpenVR initialized!"

### DISCUSSION THREAD

**Discussion by martinmoj** [Okabintaro/SubmersedVR#148] [Q&A]
the mod wont load and im guessing this is why. im using steam vr via airlink on oculus quest 2. i also have the -vrmode openvr launch argument but it didt change anything sadly

[Message:   BepInEx] BepInEx 5.4.23.2 - Subnautica (26.05.2022 17.27.04)
[Info   :   BepInEx] Running under Unity v2019.4.36.870
[Info   :   BepInEx] CLR runtime version: 4.0.30319.42000
[Info   :   BepInEx] Supports SRE: True
[Info   :   BepInEx] System platform: Bits64, Windows
[Message:   BepInEx] Preloader started
[Info   :   BepInEx] Loaded 1 patcher method from [BepInEx.Preloader 5.4.23.2]
[Info   :   BepInEx] Loaded 1 patcher method from [Tobey.Subnautica.ConfigHandler.Patcher 1.0.2.0]
[Info   :   BepInEx] Loaded 1 patcher method from [Tobey.UnityAudio.Patcher 2.0.2.0]
[Info   :   BepInEx] 3 patcher plugins loaded
[Info   :Legacy Config Handler] Applying override mode: Default
[Info   :   BepInEx] Patching [UnityEngine.CoreModule] with [BepInEx.Chainloader]
[Info   :Unity Audio Patcher] Enabling Unity Audio...
[Info   :Unity Audio Patcher] Unity Audio is already enabled, skipping patching.
[Message:   BepInEx] Preloader finished
[Message:   BepInEx] Chainloader ready
[Message:   BepInEx] Chainloader started
[Info   :   BepInEx] 4 plugins to load
[Info   :   BepInEx] Loading [SubmersedVR 0.1.6]
[Warning:SubmersedVR] Game was not started in VR. Don't load and apply SubmersedVR patches.
[Info   :   BepInEx] Loading [File Tree 1.0.0]
[Info   :   BepInEx] Loading [Subnautica Config Handler 1.0.2]
[Info   :   BepInEx] Loading [Unity Audio Patcher 2.0.2]
[Message:   BepInEx] Chainloader startup complete
[Info   : Unity Log] OpenVR initialized!
[Info   : Unity Log] Created eye textures with a "separate" layout.  The "multi-pass" stereo mode will be used.

[Info   : Unity Log] AddressablesUtility.LoadSceneAsync loading scene: StartScreen in mode Single
[Info   : Unity Log] Initializing Steam services
[Info   : Unity Log] OnQuit behaviour queued for deferred registering, platform services aren't initialized.
[Warning: Unity Log] DontDestroyOnLoad only works for root GameObjects or components on root GameObjects.
[Warning: Unity Log] Can't set camera projection to orthographic when XR is running and rendering to the main view. Camera is no longer valid to use for rendering. Please check camera projection settings of all camera components in your scene.
[Info   : Unity Log] AddressablesUtility.LoadSceneAsync loading scene: MenuEnvironment in mode Additive
[Info   : Unity Log] Loading game settings
[Info   : Unity Log] AddressablesUtility.LoadSceneAsync loading scene: XMenu in mode Additive
[Info   : Unity Log] Telemetry session started. Platform: 'Steam', UserId: '76561197960267366', SessionId: 403208359
[Info   : Unity Log] changeset 72550, QualityLevel = 2
[Info   : Unity Log] OK Running Win64
[Info   : Unity Log] Application version: 1.21.72550
[Info   : Unity Log] SystemInfo: 
 deviceModel = HP Pavilion Gaming Desktop TG01-1xxx (HP)
 deviceName = DESKTOP-LKA562O
 deviceType = Desktop
 deviceUniqueIdentifier = 54277a2df959d070e8808a52cc88c9dbc752c210
 graphicsDeviceID = 29695
 graphicsDeviceName = AMD Radeon RX 6600
 graphicsDeviceType = Direct3D11
 graphicsDeviceVendor = ATI
 graphicsDeviceVendorID = 4098
 graphicsDeviceVersion = Direct3D 11.0 [level 11.1]
 graphicsMemorySize = 8147
 graphicsShaderLevel = 50
 npotSupport = Full
 maxTextureSize = 16384
 operatingSystem = Windows 10  (10.0.19045) 64bit
 processorCount = 12
 processorFrequency = 3693
 processorType = AMD Ryzen 5 4600G with Radeon Graphics
 supportedRenderTargetCount = 8
 supports3DTextures = True
 supportsAccelerometer = False
 supportsComputeShaders = True
 supportsGyroscope = False
 supportsImageEffects = True
 supportsInstancing = True
 supportsLocationService = False
 supportsRawShadowDepthSampling = True
 supportsRenderTextures = True
 supportsRenderToCubemap = True
 supportsShadows = True
 supportsSparseTextures = True
 supportsStencil = 1
 supportsVibration = False
 systemMemorySize = 15748

[Info   : Unity Log] Saving game settings...
[Info   : Unity Log] Saving game settings complete!

**Comment by markballah-hue** [comment]
I have this same issue and can't find anything on how to fix it. I tried soo many things recommended but nothing works. It still says i'm not starting in VR but I am! Did you ever figure this out?

**Comment by Demones4** [comment]
I had a problem of VR not starting at all since 2025 update, the problem was that EpicGames released update which cut down VR feature.
If that was your case too...then :
to make  EpicGames Subnautica VR again
hope you have a copy of older Subnautica version, which worked in VR, if you do:
/Subnautica/Subnautica_Data/Managed/
copy contents from old Subnautica into new, when asked if overwrite - Skip these files
	
/Subnautica/Subnautica_Data/Plugins
all files from old Subnautica "Plugins" Folder into new "Plugins/x86_64/" Folder, when asked if overwrite - Skip these files
that's all, start your subnautica in vr as you did before

**Reply by MIRAZUKY** [reply]
Thanks for the information. Transferring the following files: openvr_api.dll, OVRGamepad.dll, OVRPlugin.dll from the old version of the game helped me to work with VR. The files are located: \Subnautica\Subnautica_Data\Plugins\x86_64.
